src/components/forms/NetworkDevicesForm/NetworkDeviceIPAddress.tsx

```diff
@@ -0,0 +1,93 @@
+import { type FC } from "react";
+import type { InstanceAndProfileFormikProps } from "../instanceAndProfileFormValues";
+import { Input } from "@canonical/react-components";
+import { type LxdNetwork } from "types/network";
+import { type LxdNicDevice } from "types/device";
+import { typesWithNicStaticIPSupport } from "util/networks";
+
+interface Props {
+  formik: InstanceAndProfileFormikProps;
+  index: number;
+  network?: LxdNetwork;
+  device: LxdNicDevice;
+  readOnly: boolean;
+  family: "IPv4" | "IPv6";
+}
+
+export const NetworkDeviceIPAddress: FC<Props> = ({
+  formik,
+  index,
+  network,
+  device,
+  readOnly,
+  family,
+}) => {
+  if (!network || !network.config) {
+    return null;
+  }
+
+  const getIPAddressString = () => {
+    return family === "IPv4" ? "ipv4.address" : "ipv6.address";
+  };
```

Comment:
```suggestion
  const addressString = family === "IPv4" ? "ipv4.address" : "ipv6.address";
```
Consider using directly evaluated `const`, it seems shorter and simpler than defining callbacks.